Imager.js

```diff
@@ -142,30 +143,36 @@
           });
         }
 
-        if (elements) {
-            this.divs = applyEach(elements, returnFn);
-            this.selector = null;
-        }
-        else {
-            this.divs = applyEach(doc.querySelectorAll(this.selector), returnFn);
-        }
-
+        this.divs = [];
+        this.add(elements || this.selector);
         this.ready(opts.onReady);
-        this.changeDivsToEmptyImages(this.divs);
 
-        setTimeout(function(){
+        setTimeout(function () {
             self.init();
         }, 0);
-    }
+    };
+
+    Imager.prototype.add = function (elementsOrSelector) {
+
+        elementsOrSelector = elementsOrSelector || this.selector;
```

Comment:
> having a mutating function without parameters seems odd.

Yes agree with this.

> a call to .add() will rematch and add existing responsive images in the pool – if you call it 5 times without adding any new image, the imgr.divs length will be multiplied by 5 anyway

Doesn't the matching selector class is dropped from transformed elements i.e.

```
<div class='delayed-image-load' data-src='...'></div>
```

To...
```
<img class='image-replace' src='...'>
```

So calling ```add()``` multiple times won't increase the size of the ```imgr.div``` array.